src/nvim/fileio.c

```diff
@@ -2756,6 +2758,18 @@ buf_write(
          * Isolate one directory name, using an entry in 'bdir'.
          */
         (void)copy_option_part(&dirp, IObuff, IOSIZE, ",");
+        p = IObuff + STRLEN(IObuff);
+        if (after_pathsep((char *)IObuff, (char *)p) && p[-1] == p[-2]) {
+          // Ends with '//', Use Full path
+          if ((p = (char_u *)make_percent_swname((char *)IObuff,
+                                                 (char *)fname)) != NULL) {
+            backup = (char_u *)modname((char *)p,
```

Comment:
Don't break lines for each argument. Try to fit as much arguments as possible on a line unless you exceed 80 characters.

(Same for the other occurrences below.)

Reply:
fixed, thanks!